### PR TITLE
tests: Improve e2e dev test case

### PR DESCRIPTION
**Pull request by Finistere**
The update mutation was returning stale data before, improving a e2e
test to check for it.

# Description

Please include a summary of the change and which issue is fixed.
Please also include relevant motivation and context.

# Type of change

- [ ] 💔 Breaking
- [ ] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [ ] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

File: cli/crates/cli/tests/graphql/default/create.graphql [hunk 1]
```diff
@@ -3,17 +3,20 @@ mutation {
     input: {
       title: "My todo list"
       todos: [
-        { create: { title: "My first todo!", complete: true } }
-        { create: { title: "My second todo!", complete: false } }
+        {create: {title: "My first todo!", complete: true}}
+        {create: {title: "My second todo!", complete: false}}
```
Comment (mlafeldt) [inline cli/crates/cli/tests/graphql/default/create.graphql:7]: Those extra spaces are added by prettier (there's a config in the root folder). Let's stick to the formatter.

**Review by mlafeldt — APPROVED** [review]
LGTM. Just a formatting nit.

File: cli/crates/cli/tests/graphql/default/create.graphql (hunk 1 above)
Comment (yoav-lavi) [reply cli/crates/cli/tests/graphql/default/create.graphql:7]: We need to add it to the CI but yes, we should be using prettier

Comment (Finistere) [reply cli/crates/cli/tests/graphql/default/create.graphql:7]: Ok, fixing it! thx

**Review by yoav-lavi — APPROVED** [review]
Looks good!

**Pull request merged**